pkg/operation/shoot/shoot.go

```diff
@@ -129,6 +129,12 @@ func (s *Shoot) GetIngressFQDN(subDomain string) string {
 	return fmt.Sprintf("%s.%s.%s", subDomain, common.IngressPrefix, *(s.Info.Spec.DNS.Domain))
 }
 
+// GetAPIServerDomain returns the fully qualified domain name of for the api-server for the Shoot cluster. The
+// end result is 'api.<domain>'.
+func (s *Shoot) GetAPIServerDomain(domain string) string {
```

Comment:
Any reason why you use a pointer receiver `s *Shoot` here?

Reply:
Thanks, good point. Moved to `common` package.